src/main/resources/applicationContext.xml

```diff
@@ -20,5 +20,5 @@
 	<bean id="mongoTemplate" class="org.springframework.data.mongodb.core.MongoTemplate">
 		<constructor-arg ref="mongoDB"></constructor-arg>
 	</bean>
-
+	
 </beans>
```

Comment:
why are we creating beans in xml file...we thought to use spring 4.0 (annotations).